$OpenBSD: patch-configure,v 1.22 2010/04/20 19:11:42 ajacoutot Exp $

The only reason it needs a newer Glib is for
g_desktop_app_info_get_filename() which we have backported in our
current glib2 port.

--- configure.orig	Tue Apr 20 21:09:09 2010
+++ configure	Tue Apr 20 21:09:30 2010
@@ -18780,12 +18780,12 @@ if test -n "$BASE_DEPENDENCIES_CFLAGS"; then
     pkg_cv_BASE_DEPENDENCIES_CFLAGS="$BASE_DEPENDENCIES_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_BASE_DEPENDENCIES_CFLAGS=`$PKG_CONFIG --cflags "glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>/dev/null`
+  pkg_cv_BASE_DEPENDENCIES_CFLAGS=`$PKG_CONFIG --cflags "glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -18796,12 +18796,12 @@ if test -n "$BASE_DEPENDENCIES_LIBS"; then
     pkg_cv_BASE_DEPENDENCIES_LIBS="$BASE_DEPENDENCIES_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_BASE_DEPENDENCIES_LIBS=`$PKG_CONFIG --libs "glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>/dev/null`
+  pkg_cv_BASE_DEPENDENCIES_LIBS=`$PKG_CONFIG --libs "glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -18819,14 +18819,14 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        BASE_DEPENDENCIES_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>&1`
+	        BASE_DEPENDENCIES_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>&1`
         else
-	        BASE_DEPENDENCIES_PKG_ERRORS=`$PKG_CONFIG --print-errors "glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>&1`
+	        BASE_DEPENDENCIES_PKG_ERRORS=`$PKG_CONFIG --print-errors "glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$BASE_DEPENDENCIES_PKG_ERRORS" >&5
 
-	as_fn_error "Package requirements (glib-2.0 >= 2.23.6    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6) were not met:
+	as_fn_error "Package requirements (glib-2.0 >= 2.22.5    atk >= 1.29.2    pango >= 1.20    cairo >= 1.6) were not met:
 
 $BASE_DEPENDENCIES_PKG_ERRORS
 
@@ -20316,7 +20316,7 @@ fi
     PKG_CONFIG=no
   fi
 
-  min_glib_version=2.23.6
+  min_glib_version=2.22.5
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for GLIB - version >= $min_glib_version" >&5
 $as_echo_n "checking for GLIB - version >= $min_glib_version... " >&6; }
 
@@ -20503,7 +20503,7 @@ rm -f core conftest.err conftest.$ac_objext \
      GOBJECT_QUERY=""
      GLIB_MKENUMS=""
      as_fn_error "
-*** GLIB 2.23.6 or better is required. The latest version of
+*** GLIB 2.22.5 or better is required. The latest version of
 *** GLIB is always available from ftp://ftp.gtk.org/pub/gtk/." "$LINENO" 5
   fi
 
@@ -20516,7 +20516,7 @@ rm -f core conftest.err conftest.$ac_objext \
 
 # See if it's safe to turn G_DISABLE_DEPRECATED on.
 GLIB_VERSION_MAJOR_MINOR=`$PKG_CONFIG --modversion glib-2.0 | sed "s/\.[^.]*\$//"`
-GLIB_REQUIRED_VERSION_MAJOR_MINOR=`echo 2.23.6 | sed "s/\.[^.]*\$//"`
+GLIB_REQUIRED_VERSION_MAJOR_MINOR=`echo 2.22.5 | sed "s/\.[^.]*\$//"`
 if test "x$GLIB_VERSION_MAJOR_MINOR" = "x$GLIB_REQUIRED_VERSION_MAJOR_MINOR"; then
   CFLAGS="-DG_DISABLE_DEPRECATED $CFLAGS"
 fi
